assets/gulpfile.js

```diff
@@ -0,0 +1,20 @@
+const { src, dest, series } = require('gulp');
+const del = require('del');
+
+const clean = () => {
+  return del(['./dist/*'], { force: true });
+};
+
+const moveFonts = () => {
+  return src(['./src/fonts/**'])
+  .pipe(dest('./dist/fonts'));
+};
+
+const moveImages = () => src(['./src/images/**'])
+.pipe(dest('./dist/images'));
+
+const moveScss = () => src(['./src/scss/**/*.scss'])
+.pipe(dest('./dist/scss'));
+
+exports.default = series(clean, moveFonts, moveImages, moveScss);
```

Comment:
Since we are moving the whole src folder to dist, can't we just 
```
const moveAssets = () => {
  return src(['./src/**'])
  .pipe(dest('./dist'));
};
```